src/v/reflection/async_adl.h

```diff
@@ -84,4 +131,45 @@ struct async_adl<std::vector<T>> {
           });
     }
 };
+
+/// Specializations for most types of maps used throughout the codebase
+template<typename K, typename V, typename KeyCmp, typename Alloc>
```

Comment:
I don't expect these things to change at this point, but implementations are free to add additional defaulted parameters, it might make sense to have a variadic parameter pack at the end,

Reply:
Great point, implemented